x/authority/genesis.go

```diff
@@ -11,6 +11,10 @@ import (
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	k.SetPolicies(ctx, genState.Policies)
 	k.SetChainInfo(ctx, genState.ChainInfo)
+	err := k.SetAuthorizationList(ctx, genState.AuthorizationList)
+	if err != nil {
+		ctx.Logger().Error("Failed to set authorization list in InitGenesis", "error", err)
```

Comment:
We should panic if initGenesis fails

Reply:
1fd660e5fbf39783b37daa8c0d4e2a4c356a0aa7